examples/web3.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useCallback } from 'react';
+
+import {
+  useEthereumProvider,
+  EthereumOnboardingOptions,
+  PlaidWeb3OnSuccess,
+  EIP1193Provider,
+} from 'react-plaid-link/web3';
```

Comment:
nice, love how clean the import is when using a completely separate top level dir (even if it is a bit janky in the build / gitignore config)